### PR TITLE
OTL-1409 Splunk Otel  Chart & Operator Auto Instrumentation

### PR DESCRIPTION
**Pull request by jvoravong**
Starting a draft review for a more private preview. Feel free to comment as needed. 

I need to fix the "make render" action to support the examples included in this PR. I'm going to do this in a separate PR. After this render issue is fixed, I'll be able to included rendered manifests for the end result for evaluation.

### CODE REVIEW

File: examples/render-examples.sh [hunk 1]
```diff
@@ -1,52 +1,52 @@
-#!/usr/bin/env bash
```
Comment (atoulme) [inline examples/render-examples.sh]: I guess this was commented out at a time - please uncomment?

Comment (jvoravong) [reply examples/render-examples.sh]: I didn't make this render script able to handle multiple yaml files in one examples dir, this issue can break several things. Sorry I was only trying to disable this while I get the fix needed out in another PR.

**Comment by jinja2** [comment]
@jvoravong does the upstream chart have an alternative option to generate a self-signed cert for the webhook instead of having to deploy cert-manager? I think k8s admins not already using cert-manager might find installing and maintaining it just for a single admission webhook a bit  heavy-handed.

**Comment by jvoravong** [comment]
> @jvoravong does the upstream chart have an alternative option to generate a self-signed cert for the webhook instead of having to deploy cert-manager? I think k8s admins not already using cert-manager might find installing and maintaining it just for a single admission webhook a bit heavy-handed.

There are some options.
https://github.com/open-telemetry/opentelemetry-helm-charts/blob/69587a482905c2c7066575a4cf30eb8095a9f2a5/charts/opentelemetry-operator/values.yaml#L145

**Pull request closed without merging**